processing/src/test/java/org/apache/druid/segment/TestHelper.java

```diff
@@ -484,6 +484,17 @@ public static Map<String, Object> makeMap(boolean explicitNulls, Object... vals)
     return theVals;
   }
 
+  public static Map<String, Object> makeMapWithExplicitNull(Object... vals)
```

Comment:
for some reason the test was complaining at me about ambiguous method when trying to use ```makeMap(boolean explicitNulls, Object... vals)``` so i added this version

Reply:
My guess: the JVM doesn't like to statically put `null` into an Object[].  You can try, perhaps calling `makeMap(true, new Object[]{1234L, "x", null}...)`

Or this is fine too.

Reply:
ah yeah, that makes sense, but seems like it somewhat defeats the purpose of having the flag. Maybe i should just transition callers to use the renamed one and then its never ambiguous